### DISCUSSION TITLE
18차 모임(6/22) 발표 주제

### DISCUSSION THREAD

**Discussion by haksungjang** [OpenChain-KWG/meeting#2] [General]
18차 OpenChain Korea Work Groupu 정기 모임이 2023년 6월 22일에 있습니다. 

- 이 때 주제 발표를 원하시는 분은 발표 제목과 간단한 내용을 comment로 남겨주시기 바랍니다. 
- 혹시, (내가 발표하지는 못하지만) 누군가가 공유해주기를 원하시는 주제가 있어도 comment 남겨주세요. 

감사합니다.

**Comment by haksungjang** [comment]
삼성전자, 카카오, LG전자와 정부가 함께하는 ['오소리' 프로젝트 출범 소식](https://www.ipdaily.co.kr/2023/04/27/11/28/38/25316/%EC%95%88%EC%A0%84%ED%95%9C-%EC%98%A4%ED%94%88%EC%86%8C%EC%8A%A4-sw-%ED%99%9C%EC%9A%A9%EC%9D%84-%EC%9C%84%ED%95%9C-%EC%98%A4%EC%86%8C%EB%A6%ACosori-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8/)을 들었습니다. 

어떤 프로젝트이며, 국내 기업이 어떻게 활용할 수 있을지 등 자세한 내용이 궁금합니다. @정윤환

**Reply by haksungjang** [reply]
@정윤환 님께서 발표해주기로 하셨습니다. 감사합니다.

**Comment by haksungjang** [comment]
LG전자가 글로벌 제조업체 최초로 오픈소스 소프트웨어 보안 관리체계 준수기업으로 [인정받았다는 기사](https://live.lge.co.kr/lg-openchain/)를 보았습니다. 
어떻게 이런 성과를 달성했는지, 다른 기업도 이러한 활동을 수행하기 위해서는 어떤 준비가 필요한지 등에 대한 설명을 듣고 싶습니다. :) @k2heart

**Reply by k2heart** [reply]
security팀에 요청해서 발표해주시기로 했습니다. 
구체적인 발표자/제목 등은 추후 정리해서 전달드리겠습니다.

**Reply by haksungjang** [reply]
LG전자 security팀 일정 상, 이번 모임은 어렵고, 다음 모임에서 발표해주기로 하셨습니다. 감사합니다.

**Comment by jungsp** [comment]
저 발표신청합니다. 이번에 Copilot 소송 판결이 나왔는데, 이걸 주제로 다뤄보도록 하겠습니다. 
가제목: "Copilot 소송 판결문 분석"

일단 이렇게 하고, 혹시 더 자극적인 제목이 떠오르면 바꿔보도록 하겠습니다. 감사합니다.^^

**Reply by haksungjang** [reply]
더 자극적인 제목: 'GitHub Copilot 써도 법적으로 문제 없나요?' ㅎㅎ

**Reply by haksungjang** [reply]
'전산학 박사 출신 특허법무대학원생이 파헤친 Copilot 소송 판결문' (30 min) / 박정숙 님 (ETRI)

**Comment by soimkim** [comment]
LG에서 FOSSLight + Security Release 주제로 발표하고자 합니다! (CC: @k2heart )

**Reply by haksungjang** [reply]
넵, 감사합니다. 발표자는 어느분 이신가요?

**Reply by soimkim** [reply]
석지영 @dd-jy 님이 발표 예정입니다. 😄

**Comment by haksungjang** [comment]
금융결제원 유대열 님께 '금융권 폐쇄망에서 OLIVE로 오픈소스 관리하기' 란 주제로 발표를 요청드렸습니다. 
다만, 유대열 님 일정 상 이번 모임에서는 어렵고, 3분기 혹은 4분기에서 소개가 가능할 것 같다는 답변을 받았습니다. 
감사합니다. :)
